.github/workflows/dependabot-auto-merge.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     if: ${{ github.event.pull_request.user.login == 'dependabot[bot]' }}
     steps:
       - name: Fetch metadata
-        uses: dependabot/fetch-metadata@v1.4.0
```

Comment:
The other option since we control this action is to pin `@v1`... but since this ships to GHES I'm not sure if it might be better to keep it fully deterministic in the off-chance that we accidentally ship a breaking change to `fetch-metadata` w/o realizing it we don't hose all those GHES users... 🤷‍♂️